### PR TITLE
Prepare to release oav version 0.4.2

### PR DESCRIPTION
**Pull request by vishrutshah**
Fixes: https://github.com/Azure/oav/issues/122
+ 
Preparing for the new release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,9 +54,19 @@ npm install -g oav
 ```
 bash-3.2$ oav -h
 Commands:
-  validate-example <spec-path>  Performs validation of x-ms-examples and
-                                examples present in the spec.
-  validate-spec <spec-path>     Performs semantic validation of the spec.
+  extract-xmsexamples <spec-path>           Extracts the x-ms-examples for a
```
Comment (veronicagg) [inline README.md]: I wonder if we should re-order to show the validate commands that we expect people to use most, first.

File: README.md [hunk 2]
```diff
@@ -54,9 +54,19 @@ npm install -g oav
...
+  <recordings>                              given swagger from the .NET session
+                                            recordings and saves them in a file.
+  generate-wireformat <spec-path>           Transforms the x-ms-examples for a
+                                            given operation into raw
+                                            request/response format and saves
+                                            them in a markdown file.
+  resolve-spec <spec-path>                  Resolves the swagger spec based on
```
Comment (veronicagg) [inline README.md]: the explanation for this one may need more details, I'm not sure I get what it does or why I would use it from the text there. Do we need more documentation on some of these?

Comment (vishrutshah) [inline README.md]: How about `Resolves the swagger spec based on the selected options like allOfs, relativePaths, examples etc.` ?

File: README.md (hunk 1 above)
Comment (vishrutshah) [inline README.md]: Currently by default it does sorting alphabetically. Quickly looking at [commandDir](https://github.com/yargs/yargs#commanddirdirectory-opts) I do not see the straight forward option yet. May be i'll just leave it for now.

**Comment by vishrutshah** [comment]
Thanks for feedback and review!

**Pull request merged**